compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/ConvertConv2DToWinograd.cpp

```diff
@@ -59,7 +62,10 @@ static bool hasValidStridesAndDilations(Operation *op) {
   return true;
 }
 
-static bool isValidConv2d(Operation *op, bool &isNchw) {
+static bool isValidConv2d(Operation *op, bool &isNchw, bool ignoreAnnotations) {
+  if (!ignoreAnnotations && !op->hasAttr("winograd_conv")) {
```

Comment:
The simplest way to do this would be with two passes. Instead of adding the load + call to the transform interpreter here, it would be easiest to just use the `--iree-preprocessing-transform-spec-filename` to do the annotation and then resolve it here.